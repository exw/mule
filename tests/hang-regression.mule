4 : all a. a
