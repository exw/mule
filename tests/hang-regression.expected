
Desugared:

	(App ((WithType (Quant (() All a (Var (() a))))) (Integer 4)))
Type error: permission error during graft
